src/planning/react/US2/es/Task2.md

```diff
@@ -0,0 +1,43 @@
+# Implementar cargador de configuración para archivo .env
+
+## Descripción:
+
+Desarrollar un cargador de configuración que lea el archivo _.env_ y haga que el token de API esté disponible dentro de la aplicación.
+
+Esta tarea garantiza que la aplicación esté equipada con un cargador de configuración robusto, que permita un acceso fluido al token de API especificado en el archivo _.env_ en toda la base de código. Los mecanismos de gestión de errores también contribuyen a la fiabilidad del proceso de carga de la configuración.
+
+## Criterios de aceptación:
+
+- [ ] Cargador de configuración:
+
+    - [ ] Implementar una función/módulo cargador de configuración que lea el fichero _.env_ y cargue sus pares clave-valor en la aplicación.
+
+    - [ ] Usar una librería como _dotenv_ para manejar la carga de variables de entorno.
```

Comment:
Se puede usar `VITE_` como se describe [acá](https://vitejs.dev/guide/env-and-mode#env-variables)?

Reply:
Si. Voy a eliminar esta tarea entonces.